scene/debugger/scene_debugger.cpp

```diff
@@ -99,6 +99,24 @@ void SceneDebugger::_handle_input(const Ref<InputEvent> &p_event, const Ref<Shor
 	}
 }
 
+void SceneDebugger::_handle_embed_input(const Ref<InputEvent> &p_event, const Dictionary &p_settings) {
+	Ref<InputEventKey> k = p_event;
+	Array keys = p_settings.get("editor/suspend_resume_embedded_project", Array()).operator Array();
+
+	Ref<Shortcut> p_shortcut = DebuggerMarshalls::deserialize_key_shortcut(keys);
```

Comment:
You should deserialize the shortcut once, instead on every input.

Reply:
That makes a lot of sense. I'm deserializing all the shortcuts before passing it to the input now.